### PR TITLE
Nanites port

### PR DESCRIPTION
**Pull request by silicons**
partially /tg/ mirrors some affected files as staging for later.

### CODE REVIEW

**Comment by silicons** [comment]
alright this is ready

**Comment by deathride58** [comment]
This PR gave the entire station diabetes. Congrats. You turned the server into America.

File: code/modules/research/nanites/nanite_programs/buffing.dm [hunk 1]
```diff
@@ -12,6 +12,12 @@
 		var/mob/living/carbon/human/H = host_mob
 		H.physiology.stun_mod *= 0.5
 
+/datum/nanite_program/nervous/active_effect()
+	. = ..()
+	if(iscarbon(host_mob))
+		var/mob/living/carbon/C = host_mob
+		C.adjustStaminaLoss(-5)
```
Comment (deathride58) [inline code/modules/research/nanites/nanite_programs/buffing.dm]: This is pretty strong for the cost.

**Comment by Poojawa** [comment]
Considering the second hand feedback, this is blatantly broken and overpowered. There's not even a lick of balance to it either.

**Review by deathride58 — APPROVED** [review]

**Pull request merged**